### DISCUSSION TITLE
Add account verification and security options to mobile app

### DISCUSSION THREAD

**Discussion by scryptoginger** [community/community#135696] [Mobile]
I have been locked out of this account (on browser) for over a month, waiting for Support to respond to my ticket for help. I cannot reset my password because the email address I use to sign in apparently isn't "verified". If there were recovery options available within the mobile app, or at least the option to manage verified email addresses, I could resolve this myself because I still have access on the mobile app. 

Please make the mobile app better in this way!!

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.172.0 (138201153)
Device: iPhone 11 (iOS 17.5.1)
</details>

**Comment by taki-on** [comment]
@scryptoginger I am sorry to hear that you are blocked on the Web. Mobile app is an OAuth app, and your oauth token is still valid, so that you can access to GitHub via GitHub Mobile, I believe. Unfortunately, we do not have a concrete plan to support managing email verification or recovery options in the app. I hope our support team can unblock you soon!

**Reply by shheyyy** [reply]
Hello I am logged into git hub with mmmollym1985@hotmail.ca credentials however cannot access my hotmail. Also, this email is linked to several accounts and my phone number. Also used for apple with same I'd. please help
